DeSci page collaborative update with much more content (#6795)
ethereum/ethereum-org-website: src/content/desci/index.md

The goal of DeSci is to create an ecosystem in which scientists are incentivized to openly share their research and receive credit for their work, while also providing a way for the general public to easily access and contribute to scientific research. DeSci is built on the premise that scientific knowledge should be open and accessible to everyone, and that the process of scientific research should be transparent. DeSci is also intended to create a more decentralized and distributed model for scientific research, which would make it more resistant to censorship and control by central authorities. By decentralizing access to funding, scientific tools and communication channels, DeSci hopes to create an environment where new and unconventional ideas can flourish.

Decentralized science allows for more diverse funding sources (from [DAOs](/dao), [quadratic donations](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2003531) to crowdfunding and more), easier access to data and methods, and by providing incentives for reproducibility.

## How DeSci improves science

> An incomplemete list of key problems in science and how decentralized science can help to address these issues

Reviewer: ```suggestion
An incomplete list of key problems in science and how decentralized science can help to address these issues
```

While free and open-access platforms do exist in the form of pre-print servers (e.g. ArXiv) these platforms lack quality control and [anti-sybil](https://csrc.nist.gov/glossary/term/sybil_attack) mechanisms and do not generally track article-level metrics, meaning they are usually only used to publicize work prior to submission to a traditional publisher. SciHub also exists to make published papers free to access, but not legally, and only after the publishers have already taken their payment and wrapped the work in strict copyright legislation. This leaves an important gap for accessible science papers and data that has an embedded legitimacy mechanism and incentive model. The tools for building such a system exist in Web3.

## Reproducibility and Replicability

Reproducibility and replicability are the foundations of quality scientific discovery. Findings that are reproducible can be achieved multiple times in a row by the same team using the same methodology. Findings that are replicable can be remade by a different team using the same experimental set-up. 

> New primitives being created ensure that reproducibility and replicability are the basis of discovery. We can weave quality science into the technological fabric of academia. Web3 offers the ability to create attestations for each component of an analysis: the raw data, the computational engine, and the result of the application. The beauty of consensus systems is that when a trust network is created for maintaining these components, each network participant can be responsible for reproducing the calculation and validating each result.

Reviewer: ```suggestion
New Web-3 native tools can ensure that reproducibility and replicability are the basis of discovery. We can weave quality science into the technological fabric of academia. Web3 offers the ability to create attestations for each component of an analysis: the raw data, the computational engine, and the result of the application. The beauty of consensus systems is that when a trust network is created for maintaining these components, each network participant can be responsible for reproducing the calculation and validating each result.
```

* [Building Trustless Systems of Science by Jon Starr](https://medium.com/@jringo/building-systems-of-trustless-science-1cd2d072f673)
* [The Emergence of Biotech DAOs](https://molecule.to/blog/the-emergence-of-biotech-daos)

## Videos
* [What's Decentralized Science? ](https://www.youtube.com/watch?v=-DeMklVWNdA)
* [Conversation between Vitalik Buterin and the scientist Aubrey de Grey about the intersection of longevity research and crypto](https://www.youtube.com/watch?v=x9TSJK1widA)
* [Conversation between Vitalik Buterin and the scientist Aubrey de Grey about the intersection of longevity research and crypto](https://www.youtube.com/watch?v=x9TSJK1widA)
* [Scientific Publishing Is Broken. Can web3 Fix It?](https://www.youtube.com/watch?v=WkvzYgCvWj8)

### Get involved: explore projects below to join the DeSci movement.
* [Atoms: smart contract peer-review)](https://www.atoms.org)
* [dClimate API: query climate data collected by a decentralized community](https://api.dclimate.net/)
* [DeSci Foundation: DeSci publishing tool builder](https://descifoundation.org/)

> * [DeSci.World): one-stop shop for users to view, engage with decentralized science) 

Reviewer: ```suggestion
* [DeSci.World: one-stop shop for users to view, engage with decentralized science](desci.world) 
```

sidebarDepth: 2
image: ../../assets/future_transparent.png
alt: TODO.

> summaryPoint1: An global, open alternative to the current scientific system.

Reviewer: ```suggestion
summaryPoint1: A global, open alternative to the current scientific system.
```

summaryPoint2: Technology that let yous raise funding, run experiments, share data, distribute insights, and more.

> summaryPoint3: Based on the open science movement.

Reviewer: ```suggestion
summaryPoint3: Builds on the open science movement.
```

## The problems in science

Science has been failing to keep up with the demands of the modern world for centuries. The mechanisms that have governed scientific inquiry for centuries are no longer providing all the necessary advancements needed in medicine, energy production, materials, exploration, mathematics, and other areas of scientific development. 

This is due in part to the gatekeeping of knowledge and resources by third party arbiters, as well as the inability of the general public to participate in and understand the scientific process. 

> Decentralized science can help to address these issues by allowing for more diverse funding sources (from DAOs, quadratic donations to crowdfunding and more), easier access to data and methods, and by providing incentives for reproducibility.

Reviewer: ```suggestion
Decentralized science can help to address these issues by allowing for more diverse funding sources (from [DAOs](/dao), [quadratic donations](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2003531) to crowdfunding and more), easier access to data and methods, and by providing incentives for reproducibility.
```

* poor data quality -> incentivising transparent data sharing and quality control. 
* [publication bias](https://forrt.org/glossary/publication-bias-file-drawer-proble/) -> enabling publication in a wider range of outlets. 
* [p-hacking](https://www.methodspace.com/blog/primer-p-hacking) -> incentivising transparent and honest reporting of results.
* [cherry picking](https://pubmed.ncbi.nlm.nih.gov/33999541/) -> requiring publication of all results, including negative or inconclusive ones. 
* [HARKing](https://forrt.org/glossary/harking/): hypothesizing after the results are known -> providing incentives for transparent and honest reporting of results.

A comparison to traditional science

<table>
  <tr>
   <td><strong>Decentralized science (aspiration)</strong>
   </td>
   <td><strong>Traditional science</strong>
   </td>
  </tr>
  <tr>
   <td>Distribution of funds is determined by the public using mechanisms such as quadratic voting.
   </td>
   <td>Distribution of funds is controlled by small, closed, centralized groups.	
   </td>
  </tr>
  <tr>
   <td>You collaborate with peers from all over the globe in dynamic teams.
   </td>

>    <td>Your collaborations are limited by funding organizations and home institutions.

Reviewer: ```suggestion
   <td>Collaborations are often from smaller networks of familiar participants.
```

   </td>
  </tr>
  <tr>

>    <td>Funding decisions are made online and in a transparent way. New funding mechanisms are explored.

Reviewer: ```suggestion
   <td>Funding decisions are made online and in a transparent way. New funding and voting mechanisms are available.
```

   </td>
   <td>Funding decisions are made with a long turnaround time and limited transparency. Few funding mechanisms exist.
   </td>
  </tr>
  <tr>
   <td>DeSci is open to anyone.
   </td>
   <td>You must be a member or a physical research institute to do science.
   </td>
  </tr>
  <tr>
   <td>You can trade laboratory services freely.
   </td>
   <td>You are constrained to using your local institute's laboratory infrastructure.
   </td>
  </tr>
  <tr>

>    <td>You publish your work online on your own terms.

Reviewer: ```suggestion
   <td>New models for publishing can be developed that use Web3 primitives for trust, transparency and universal access.
```

   </td>
   <td>You publish through established pathways that are frequently acknowledged to be inefficient, biased and exploitative.
   </td>
  </tr>
  <tr>
   <td>You can earn tokens and reputation for peer-reviewing work.
   </td>
   <td>Your peer-review work is unpaid, benefiting for-profit publishers.
   </td>
  </tr>
  <tr>

>    <td>You own the IP you generate and distribute it according to transparent terms.

Reviewer: ```suggestion
   <td>You own the intellectual property (IP) you generate and distribute it according to transparent terms.
```

**Contributors to Web3**

The people and institutions who have been brought on board through DeSci's tooling have a lot of experience and knowledge in different areas. They've been working on similar problems to those facing Web3 for a long time. DeSci is making tools that will make their lives easier and let them keep working on the hardest and most rewarding problems facing society.  Having economists and mathematicians, to social psychologists and historians, with centuries of knowledge from hundreds of thousands of academics can help build the foundation of the macro-systems we're trying to create. In doing this, we're inviting them into the trustless world to keep doing what they're good at: solving the most challenging and fulfilling problems society has to offer.

## Innovation

> Decentralized science will require new ways of funding, creating, reviewing, crediting, storing, and disseminating knowledge. These novel developments will undoubtedly affect other realms as well. From rich on-chain research objects to IP-NFTs and everything in between, the development happening in DeSci will be pivotal to the digital world moving forward.

Reviewer: ```suggestion
Decentralized science will require new ways of funding, creating, reviewing, crediting, storing, and disseminating knowledge. These novel developments will undoubtedly affect other realms as well. From rich on-chain research objects to IP recorded as [NFTs](/nft) and everything in between, the development happening in DeSci will be pivotal to the digital world moving forward.
```

## Legitimacy

The crypto world is threatened by the possibility of extreme regulation. Web3 needs examples of blockchain use-cases that are easily understood and palatable, demonstrating a capability to increase efficiency, efficacy, reliability, and public trust of critical systems. DeSci is perhaps the best positioned to develop these examples..

## Scalable knowledge creation

Current research on Web3 is conducted in a way that is not efficient or effective in producing reliable information. The scientific method is a much better way to distill knowledge, and academia is trained in using this method. DeSci is creating the tool stack needed to apply the scientific method to research on Web3.

# DeSci use cases

DeSci is building the scientific toolstack needed to onboard Web2 academia into the digital world. Below is a sampling of use cases that Web3 can offer to the scientific community. 

## Publishing

> Science publishing is famously problematic in that it is managed by publishing houses that rely upon free labor from scientists, reviewers and editors to generate the papers, but then also charge extortionate publishing fees. The public, who have usually indirectly paid for the work and the publication costs through taxation, can often not access that same work without paying the publisher again. The total costs for publishing individual science papers is often five-figures ($USD). This undermines the whole concept of scientific knowledge as a public good while generating huge profits for a small group of publishers.

Reviewer: ```suggestion
Science publishing is famously problematic in that it is managed by publishing houses that rely upon free labor from scientists, reviewers and editors to generate the papers, but then also charge extortionate publishing fees. The public, who have usually indirectly paid for the work and the publication costs through taxation, can often not access that same work without paying the publisher again. The total costs for publishing individual science papers is often five-figures ($USD). This undermines the whole concept of scientific knowledge as a [public good](https://www.econlib.org/library/Enc/PublicGoods.html) while generating huge profits for a small group of publishers.
```

> While free and open-access platforms do exist in the form of pre-print servers (e.g. ArXiv) these platforms lack quality control and anti-sybil mechanisms and do not generally track article-level metrics, meaning they are usually only used to publicize work prior to submission to a traditional publisher. SciHub also exists to make published papers free to access, but not legally, and only after the publishers have already taken their payment and wrapped the work in strict copyright legislation. This leaves an important gap for accessible science papers and data that has an embedded legitimacy mechanism and incentive model. The tools for building such a system exist in Web3.

Reviewer: ```suggestion
While free and open-access platforms do exist in the form of pre-print servers (e.g. ArXiv) these platforms lack quality control and [anti-sybil](https://csrc.nist.gov/glossary/term/sybil_attack) mechanisms and do not generally track article-level metrics, meaning they are usually only used to publicize work prior to submission to a traditional publisher. SciHub also exists to make published papers free to access, but not legally, and only after the publishers have already taken their payment and wrapped the work in strict copyright legislation. This leaves an important gap for accessible science papers and data that has an embedded legitimacy mechanism and incentive model. The tools for building such a system exist in Web3.
```

## Funding

The current standard model for funding science is that individuals or groups of scientists make written applications to a funding agency. A small panel of trusted individuals score the applications and then interview candidates before awarding funds to a small portion of applicants. Aside from creating bottlenecks that lead to sometimes years of waiting time between applying for and receiving a grant, this model is known to be highly vulnerable to the biases, self-interests and politics of the review panel. 

Studies have shown that grant review panels do a poor job of selecting high quality projects. The same proposals given to different panels have wildly different outcomes. As funding has become more scarce it has concentrated into a smaller pool of more senior researchers with more intellectually conservative projects. The effect has been to create a hyper-competitive funding landscape, entrenching perverse incentive and stifling innovation. 

> Web3 has the potential to disrupt this broken funding model by experimenting with different incentive models developed by DAOs and web3 broadly. Retroactive public goods funding, quadratic voting and donating, DAO governance, tokenized incentive structures are some of the web3 tools that could revolutionize science funding.

Reviewer: ```suggestion
Web3 has the potential to disrupt this broken funding model by experimenting with different incentive models developed by DAOs and web3 broadly. [Retroactive public goods funding](https://medium.com/ethereum-optimism/retroactive-public-goods-funding-33c9b7d00f0c), [quadratic funding](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2003531), [DAO governance](https://www.antler.co/blog/daos-and-web3-governance) and [tokenized incentive structures](https://cdixon.org/2017/05/27/crypto-tokens-a-breakthrough-in-open-network-design) are some of the web3 tools that could revolutionize science funding.
```

## IP Ownership and Development

> Intellectual property (IP) is a big problem in traditional science. From being stuck in universities or unused in biotechs, to being notoriously hard to value. However, ownership of digital assets (such as scientific data or articles) is something Web3 does exceptionally well using NFTs.

Reviewer: ```suggestion
Intellectual property (IP) is a big problem in traditional science. From being stuck in universities or unused in biotechs, to being notoriously hard to value. However, ownership of digital assets (such as scientific data or articles) is something Web3 does exceptionally well using [NFTs](/nft).
```

The same way that NFTs can pass revenue for future transactions back to the original creator, you can establish transparent value attribution chains to reward researchers, governing bodies (like DAOs), or even the subjects whose data is collected. 

> IP-NFTs can also function as a key to a decentralized data repository of the research experiments being undertaken, and plug into NFT and DeFi financialization (from fractionalization to lending pools and value appraisal). It also allows for natively onchain entities such as DAOs like VitaDAO to hold research directly on-chain. 

Reviewer: ```suggestion
IP-NFTs can also function as a key to a decentralized data repository of the research experiments being undertaken, and plug into NFT and [DeFi](/defi) financialization (from fractionalization to lending pools and value appraisal). It also allows for natively onchain entities such as DAOs like VitaDAO to hold research directly on-chain. 
```

> 

Reviewer: ```suggestion
The advent of non-transferable ["soulbound"](https://vitalik.ca/general/2022/01/26/soulbound.html) tokens may also play an important role in DeSci by allowing individuals to prove their experience and credentials linked to their Ethereum address.
```

* [A guide to decentralized biotech by Jocelynn Pearl for a16z future](https://future.a16z.com/a-guide-to-decentralized-biotech/)
* [The case for DeSci](https://gitcoin.co/blog/desci-the-case-for-decentralised-science/)
* [Guide to DeSci](https://future.com/what-is-decentralized-science-aka-desci/)
* [Decentralized science resources](https://www.vincentweisser.com/decentralized-science)
* [Molecule’s Biopharma IP-NFTs - A Technical Description](https://molecule.to/blog/molecules-biopharma-ip-nfts-a-technical-description)
* [Building Trustless Systems of Science by Jon Starr](https://medium.com/@jringo/building-systems-of-trustless-science-1cd2d072f673)
* [The Emergence of Biotech DAOs](https://molecule.to/blog/the-emergence-of-biotech-daos)

# Get involved: **What can you do in DeSci?**
Where can people go to start participating?
* [DeSci Wiki: Overview of initiatives and projects ](https://docs.google.com/document/d/1aQC6zn-eXflSmpts0XGE7CawbUEHwnL6o-OFXO52PTc/edit#)

### Explore projects below to join the DeSci movement.
* [Atoms](https://www.atoms.org)
* [dClimate API: query climate data collected by a decentralized community](https://api.dclimate.net/)

> * [DeSci Foundation](https://descifoundation.org/)

Reviewer: ```suggestion
* [DeSci Foundation (DeSci think tank)](https://descifoundation.org/)
```

> * [GenomesDAO](https://genomes.io/)

Reviewer: ```suggestion
```

* [Gitcoin: apply for an scientific open source developer grant](https://gitcoin.co/grants)
* [Molecule: Fund and get funded for your research projects](https://discover.molecule.to/)

> * [OceanDAO](https://oceanprotocol.com/dao)

Reviewer: ```suggestion
* [OceanDAO (DAO governed science funding](https://oceanprotocol.com/dao)
```

> * [Opscientia](https://opsci.io/research/)

Reviewer: ```suggestion
* [Opscientia (open decentralized science workflows)](https://opsci.io/research/)
```

​​

## Why we need science

Science is a process that allows us to create and translate knowledge in order to improve the quality of life for humanity as a whole. It also provides solutions to some of the most complex problems we face. In order to be trustworthy, knowledge produced by science must be replicable and open to scrutiny. This process should be impartial and available to the public. Any knowledge that cannot be replicated or tested for falsifiability should not be considered as the truth.

By creating knowledge that is impartial and available to the public, we enable ourselves to make fundamental technological advancements. These advancements give us the power to more effectively navigate and manipulate our environment, leading to vast improvements in quality of life. As we continue to create impartial knowledge and make technological advancements, we find ourselves capable of managing increasingly complex systems. Today, we are confronted with species-level existential risks, such as climate change and global pandemics. Science is the only process we have that offers equitable positive outcomes.

> ## How DeSci improves science

Reviewer 2: Maybe create a "yes and" visualization. Traditional: National agency. Desci: National agency AND x,y,z

   <td>Your collaborations are limited by funding organizations and home institutions.
   </td>
  </tr>
  <tr>
   <td>Funding decisions are made online and in a transparent way. New funding mechanisms are explored.
   </td>
   <td>Funding decisions are made with a long turnaround time and limited transparency. Few funding mechanisms exist.
   </td>
  </tr>
  <tr>
   <td>DeSci is open to anyone.
   </td>
   <td>You must be a member or a physical research institute to do science.
   </td>
  </tr>
  <tr>
   <td>You can trade laboratory services freely.
   </td>
   <td>You are constrained to using your local institute's laboratory infrastructure.
   </td>
  </tr>
  <tr>
   <td>You publish your work online on your own terms.
   </td>
   <td>You publish through established pathways.
   </td>
  </tr>
  <tr>
   <td>You can earn tokens and reputation for peer-reviewing work.
   </td>
   <td>Your peer-review work is unpaid, benefiting for-profit publishers.
   </td>
  </tr>
  <tr>
   <td>You own the IP you generate and distribute it according to transparent terms.
   </td>
   <td>Your home institution owns the IP you generate. Access to the IP is not transparent.
   </td>
  </tr>
</table>

> ## Why do we need DeSci?

Reviewer 2: Proposed Change:

# Why Web3 needs DeSci
## Adoption
Decentralized science has the potential to attract new participants and contributors to web3 and to help disseminate knowledge more effectively.

The web3 community is still in its early stages, and there is a lot of work to be done in order to realize the full potential of decentralized science. However, the potential benefits are significant, and it is worth exploring how decentralized science can be used to improve the way we conduct research and share knowledge.

Participants in Web3
The tools DeSci will produce are easily understood and explained without technical jargon. For example, research objects are scientific papers that contain more information than a simple pdf. Imagine interacting with a research paper instead of reading it. Imagine seeing the code used to run simulations and running it yourself with a simple click of a button. Imagine seeing every change, evolution, and contributor to the research over its lifetime just like with Git. 

Publishing networks, as another example, allow research to be published, reviewed, elevated, and replicated without input from a controlling authority or company. Imagine never paying to access a research object. Imagine researchers getting paid directly for their work instead of through an intermediary. Imagine a marketplace of people and entities that provide review, curation, and replication services instead of an oligopoly of a few major conglomerates. 

## Contributors to Web3
The people and institutions onboarded through DeSci tooling have spent their careers and lifetimes exploring, creating, and translating knowledge from all disciplines. They often work on problems similar to those currently facing the trustless environment. Web3 needs this seemingly limitless experience to help shape the foundation of the macro-systems we are developing. 

DeSci is building the tools that will make their lives easier. In doing so, we invite them into the trustless world to continue doing what they do best: solve the most challenging and fulfilling problems society has to offer.

Reviewer: I suggest removing the entire "Why Web3 needs DeSci" section. 

First, if Web3 "needs" DeSci then we have a very big problem. The infrastructure can't rely on one of its applications - it must be the other way around.

Second. "adoption" isn't a unique benefit from DeSci - you could apply this logic to any application.

Third, I don't follow the logic in the "contributors to Web3" section - it suggests that web3 will benefit by having DeSci-pilled scientists working on the stack, but the stack exists to help those same people work on their science, not building out the stack.

I have already explained my concerns around the "legitimacy" section.

I think it is better to remove the section altogether.

> Only Ethereum has the level of security and also a (nearly) Turing-complete virtual machine that supports a rich application layer. DeSci will thrive on Ethereum Layer-2s, where costs and transaction times are very low but security is inherited from Ethereum. A decentralized system of science will require robust security, minimal monetary and temporal transaction cost, and a rich layer-2 ecosystem for application development. Ethereum provides this foundation on which DeSci can thrive.

Reviewer 2: See proposed change to line 117

**Contributors to Web3**

The people and institutions who have been brought on board through DeSci's tooling have a lot of experience and knowledge in different areas. They've been working on similar problems to those facing Web3 for a long time. DeSci is making tools that will make their lives easier and let them keep working on the hardest and most rewarding problems facing society.  Having economists and mathematicians, to social psychologists and historians, with centuries of knowledge from hundreds of thousands of academics can help build the foundation of the macro-systems we're trying to create. In doing this, we're inviting them into the trustless world to keep doing what they're good at: solving the most challenging and fulfilling problems society has to offer.

## Innovation

> 

Reviewer 2: Proposed change:

Decentralized science will require new ways of funding, creating, reviewing, crediting, storing, and disseminating knowledge. These novel developments will undoubtedly affect other realms as well. From rich on-chain research objects to IP-NFTs and everything in between, the development happening in DeSci will be pivotal to the digital world moving forward.

summaryPoint3: Based on the open science movement.
---

## What is decentralized science (DeSci)?

Decentralized science (DeSci) is a movement that aims to build public infrastructure for funding, creating, reviewing, crediting, storing, and disseminating scientific knowledge fairly and equitably using the web3 stack.

The goal of DeSci is to create an ecosystem in which scientists are incentivized to openly share their research and receive credit for their work, while also providing a way for the general public to easily access and contribute to scientific research. DeSci is built on the premise that scientific knowledge should be open and accessible to everyone, and that the process of scientific research should be transparent. DeSci is also intended to create a more decentralized and distributed model for scientific research, which would make it more resistant to censorship and control by central authorities. By decentralizing access to funding, scientific tools and communication channels, DeSci hopes to create an environment where new and unconventional ideas can flourish.

> ## The problems in science

Author: feel free to make a final rephrased pull request, but lets keep it as it for now, to keep this pull request manageable.. its hard to incorporate if not finalized

By creating knowledge that is impartial and available to the public, we enable ourselves to make fundamental technological advancements. These advancements give us the power to more effectively navigate and manipulate our environment, leading to vast improvements in quality of life. As we continue to create impartial knowledge and make technological advancements, we find ourselves capable of managing increasingly complex systems. Today, we are confronted with species-level existential risks, such as climate change and global pandemics. Science is the only process we have that offers equitable positive outcomes.

## How DeSci improves science

**List of key problems in science and how decentralized science can help to address these issues**

* bureaucratic funding structures -> allow for more diverse funding sources (quadratic donations, DAOs, crowdfunding etc.) 

> * replication crisis -> allowing for easier access to data and methods, and by providing incentives for reproducibility.

Reviewer: ```suggestion
* [replication crisis](https://forrt.org/glossary/reproducibility-crisis-aka-replicab/) -> allowing for easier access to data and methods, and by providing incentives for reproducibility.
```

* misaligned incentives -> aligning incentives more closely with the goals of science. 
* poor data quality -> incentivising transparent data sharing and quality control. 

> * publication bias -> enabling publication in a wider range of outlets. 

Reviewer: ```suggestion
* [publication bias](https://forrt.org/glossary/publication-bias-file-drawer-proble/) -> enabling publication in a wider range of outlets. 
```

> * p-hacking -> incentivising transparent and honest reporting of results.

Reviewer: ```suggestion
* [p-hacking](https://www.methodspace.com/blog/primer-p-hacking) -> incentivising transparent and honest reporting of results.
```

> * cherry picking -> requiring publication of all results, including negative or inconclusive ones. 

Reviewer: ```suggestion
* [cherry picking](https://pubmed.ncbi.nlm.nih.gov/33999541/) -> requiring publication of all results, including negative or inconclusive ones. 
```

> * HARKing: hypothesizing after the results are known -> providing incentives for transparent and honest reporting of results.

Reviewer: ```suggestion
* [HARKing](https://forrt.org/glossary/harking/): hypothesizing after the results are known -> providing incentives for transparent and honest reporting of results.
```

A comparison to traditional science

<table>
  <tr>
   <td><strong>DeSci</strong>
   </td>
   <td><strong>Traditional science</strong>
   </td>
  </tr>
  <tr>
   <td>You raise funding from a global community.
   </td>

>    <td>You raise funding from a national agency.

Reviewer: sometimes - my experience has been to attract funding more regularly from societies, private businesses, charities and international funding agencies than national funding agencies. Reducing the current landscape down to national funding agencies seems a bit narrow.

Author: good point, would from a centralized agency vs decentralized community be a better comparison?

Reviewer: IMO the value that DeSci adds to the science funding model comes from the fact that the public can influence the distribution of funds via e.g. QF or at least delegate their influence to a trusted party, rather than the decisions being concentrated entirely in very small, centralized committees --> better alignment between community values and capital allocation.

Also that Gitcoin, Uniswap etc have shown that institutions are willing to fund "public goods" generally and that this could be a new source of science funding if the downstream incentive model is organized correctly. 

Something like the following might work:

| ...| ...|
|--------------------------------------------------------------------|--------------------------------------------------------------------------|
|Distribution of funds is controlled by small, closed, centralized groups | Distribution of funds is determined by the public using mechanisms such as quadratic voting|

Reviewer 2: I agree with the table change but would change the language in the DeSci section from "is" to "can be".

A decentralized system can still have participants manifested as small closed centralized groups that control funds

   </td>
  </tr>
  <tr>
   <td>You collaborate with peers from all over the globe in dynamic teams.
   </td>

>    <td>Your collaborations are limited by funding organizations and home institutions.

Reviewer: Again, I have never experienced a funder or institution restricting my collaborations on the basis of their affiliation, nor can I recall any instances of this in my wider network- is it really a pervasive problem?

Reviewer 2: I agree. DeSci will add more options for a researcher, but it's not that the traditional system leaves or is absolutely restrictive.

   </td>
  </tr>
  <tr>
   <td>Funding decisions are made online and in a transparent way. New funding mechanisms are explored.
   </td>
   <td>Funding decisions are made with a long turnaround time and limited transparency. Few funding mechanisms exist.
   </td>
  </tr>
  <tr>
   <td>DeSci is open to anyone.
   </td>
   <td>You must be a member or a physical research institute to do science.
   </td>
  </tr>
  <tr>
   <td>You can trade laboratory services freely.
   </td>

>    <td>You are constrained to using your local institute's laboratory infrastructure.

Reviewer: again, I'm not sure this is really true - laboratory and computing resources are shared between institutions all the time. If there is a barrier it is usually monetary.

   </td>
  </tr>
  <tr>
   <td>You publish your work online on your own terms.
   </td>
   <td>You publish through established pathways.
   </td>
  </tr>
  <tr>
   <td>You can earn tokens and reputation for peer-reviewing work.
   </td>

>    <td>Your peer-review work is unpaid, and often benefitting for-profit publishers.

Reviewer: ```suggestion
   <td>Your peer-review work is unpaid, benefiting for-profit publishers.
```

## Innovation

The systems of decentralized science require new ways of funding, creating, reviewing, crediting, storing, and disseminating knowledge. These new methods are applied to science but can be used for other purposes as well. From rich on-chain research objects to IP-NFTs and everything in between, the development happening in DeSci will be pivotal to the digital world moving forward. The Cambrian explosion of projects and concepts since inception creates an environment of rapid experimentation where anything is possible.

## Legitimacy

The crypto world is threatened by the possibility of extreme regulation. Web3 needs examples of blockchain use-cases that are easily understood and palatable, demonstrating a capability to increase efficiency, efficacy, reliability, and public trust of critical systems. DeSci is perhaps the best positioned to develop these examples..

## Scalable knowledge creation

> Current research on Web3 is conducted in a way that is not efficient or effective in producing reliable information. The scientific method is a much better way to distill knowledge, and academia is trained in using this method. DeSci is creating the tool stack needed to apply the scientific method to research on Web3.

Reviewer: Something about this section doesn't sit quite right with me, and I think it is the `legitimacy` subsection feels too self serving. 

The DeSci value proposition must be that it makes science somehow more frictionless for scientists & public and the web3 stack uniquely has the right primitives to do that. If part of the value-prop of building science infra on Web3 is propping up Web3 that signals misaligned incentives to me.

Would like to hear more opinions on this.

I do, however, love the point about DeSci as a tool for applying the scientific method to its own base infrastructure.

Author: good point! would you be open to rewrite or change it?

Reviewer 2: I agree that "DeSci provides legitimacy" can be a dangerous proposition. I do not think crypto doesn't have legitimacy already, however I think legitimacy is in question in the eyes of many. Drugs, ponzi, bubbles, NFTs, memes, etc. 

Science is a system that almost everyone can get behind. Demonstrating the use case of DLT as it applies to science might convince many doubters that yes, there are interesting growing pains to this technology, but it can also make science better... more frictionless etc.

I would propose this change to the legitimacy section:

The crypto world is threatened by the possibility of extreme regulation. Web3 needs examples of blockchain use-cases that are easily understood and palatable, and that demonstrate a capability to increase efficiency, efficacy, reliability, and public trust of critical societal systems. DeSci is perhaps the best positioned to develop these examples.

Reviewer: sure - I will let it sit for a little while to see if anyone else has opinions to share first.

Reviewer: I think the other issue I have with this is that if Web3 can enjoy a legitimacy boost from the success of DeSci, the opposite is also true. DeSci is a massive risk area, because it is a nascent space that can still easily be co-opted by profiteers and shills. These are existential risks for DeSci, and there is contagion risk for wider Web3.

# DeSci use cases

DeSci is building the scientific toolstack needed to onboard Web2 academia into the digital world. Below is a sampling of use cases that Web3 can offer to the scientific community. 

## Publishing

Science publishing is famously problematic in that it is managed by publishing houses that rely upon free labor from scientists, reviewers and editors to generate the papers, but then also charge extortionate publishing fees. The public, who have usually indirectly paid for the work and the publication costs through taxation, can often not access that same work without paying the publisher again. The total costs for publishing individual science papers is often five-figures ($USD). This undermines the whole concept of scientific knowledge as a public good while generating huge profits for a small group of publishers.

> Censorship resistance probably needs to be added

Reviewer: ```suggestion

```

While free and open-access platforms do exist in the form of pre-print servers (e.g. ArXiv) these platforms lack quality control and anti-sybil mechanisms and do not generally track article-level metrics, meaning they are usually only used to publicize work prior to submission to a traditional publisher. SciHub also exists to make published papers free to access, but not legally, and only after the publishers have already taken their payment and wrapped the work in strict copyright legislation. This leaves an important gap for accessible science papers and data that has an embedded legitimacy mechanism and incentive model. The tools for building such a system exist in Web3.

## Reproducibility and Replicability

Reproducibility and replicability are the foundations of quality scientific discovery. Findings that are reproducible can be achieved multiple times in a row by the same team using the same methodology. Findings that are replicable can be remade by a different team using the same experimental set-up. 

> New primitives being created ensure that reproducibility and replicability are the basis of discovery. We can weave quality science into the technological fabric of academia. Web3 offers the ability to create attestations for each component of an analysis: the raw data, the computational engine, and the result of the application. The beauty of consensus systems is that when a trust network is created for maintaining these components, each network participant can be responsible for reproducing the calculation and validating each result.

Author: you mean mentioning specific solutions? feel free to add at the end maybe?

> New primitives being created ensure that reproducibility and replicability are the basis of discovery. We can weave quality science into the technological fabric of academia. Web3 offers the ability to create attestations for each component of an analysis: the raw data, the computational engine, and the result of the application. The beauty of consensus systems is that when a trust network is created for maintaining these components, each network participant can be responsible for reproducing the calculation and validating each result.

Reviewer: Well the sentence started by referencing some "primitives" and I was left wondering which primitives in particular are relevant. Some embedded links, perhaps to definitions or papers, would help.

   </td>
  </tr>
  <tr>

>    <td>You raise funding from a decentralized community.

Reviewer 2: Not everyone wants to get funding from a decentralized community. The good thing is that DeSci does not require that you do. I think this should highlight choice and not an absolute.
